contracts/CollectiveGovernance.sol

```diff
@@ -65,6 +65,17 @@ import "../contracts/TimeLock.sol";
 ///
 /// @dev The VoterClass is common to all proposed votes as are the project supervisors.   Individual supervisors may
 /// be configured as part of the proposal creation workflow but project supervisors are always included.
+
+error CG__VotingNotReady();
+error CG__VotingClosed();
```

Comment:
One final comment - it would be good at a minimum to include one parameter - perhaps the uint256 proposalId on these errors so they can be isolated to a specific event when they occur.